utils/scripts/ocm/ocm.py

```diff
@@ -279,13 +279,13 @@ def update_isv_cluster_info(self):
         config_file = self.configrepo_dir + "test-variables.yaml"
         with open(config_file, "r") as fh:
             data = yaml.safe_load(fh)
-        data["MONGODB"] = {}
-        data["MONGODB"]["ORG_ID"] = self.mongo_org_id
-        data["MONGODB"]["PUB_KEY"] = self.mongo_pub_key
-        data["MONGODB"]["PRI_KEY"] = self.mongo_pri_key
-        data["CRUNCHYDB"]["PUB_KEY"] = self.crunchy_pub_key
-        data["CRUNCHYDB"]["PRI_KEY"] = self.crunchy_pri_key
-        data["COCKROACHDB"]["API_KEY"] = self.crdb_api_key
+        data["MONGO"] = {}
```

Comment:
Shouldn't a similar line also exist for Crunchy and Cockroach?

Reply:
This is initialization where the data declared as list of objects, once it is declared we can add cockroach and crunchy without error. I hope it would work with data = {} as well.

Reply:
Thanks for clarifying